rclip/db.py

```diff
@@ -40,6 +50,7 @@ def ensure_tables(self):
         filepath TEXT NOT NULL UNIQUE,
         modified_at DATETIME NOT NULL,
         size INTEGER NOT NULL,
+        hash TEXT NOT NULL, 
```

Comment:
Can we create an index for this column to speed up the queries by hash?

Reply:
The indexing column in the images table of db.py has been added to keep track of which images are currently being processed during the indexing operation.